api-tests/src/test/resources/org/openmrs/module/reporting/include/ReportTestDataset-openmrs-1.9.xml

```diff
@@ -202,6 +202,8 @@
   <encounter encounter_id="10" encounter_type="6" patient_id="22" location_id="2" form_id="1" encounter_datetime="2009-10-19 00:00:00.0" creator="1" date_created="2008-08-19 16:34:40.0" voided="false" void_reason="" uuid="c3eb9bac-c4cf-11de-a970-6ab447468ce9"/>
```

Comment:
@ivange94 if we are going to edit the ReportTestDataset, we'll need to make sure we change all of them consistently.  There are 5 different versions (activated by different build profiles), and we will need the unit tests to pass for all of them.  An alternative would be for you to add in a few test patients, test locations, test encounters, and test enrollments in a separate test data set (or in code using a helper class).  This would also ensure that any downstream modules that rely on this test data won't be impacted by adding additional patients that they are not expecting.